vllm/model_executor/models/plamo3.py

```diff
@@ -153,13 +153,23 @@ def __init__(self, *, vllm_config: VllmConfig, prefix: str = "", **kwargs) -> No
             quant_config=quant_config,
             prefix=f"{prefix}.o_proj",
         )
+
         layer_idx = extract_layer_index(prefix)
-        full_attn = config.interleaved_sliding_window[layer_idx] is None
+        layer_type = config.layer_types[layer_idx]
+        is_sliding = layer_type == "sliding_attention"
+
+        # Initialize the rotary embedding.
+        if layer_type in config.rope_parameters:
+            # Transformers v5 rope config.
+            rope_parameters = config.rope_parameters[layer_type]
+        else:
+            # Transformers v4 rope config.
+            # Global attention. Use the values in config.json.
+            rope_parameters = config.rope_parameters.copy()
```

Comment:
Making the previous change should make this copy unnecessary

```suggestion
            rope_parameters = config.rope_parameters
```